package/yast2.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Thu Mar  5 13:00:43 UTC 2020 - David Diaz <dgonzalez@suse.com>
+
+- Do not crash when reading the services database in a system
```

Comment:
The entry looks too wordy to me. What about something more concise like the following example?

*Read the list of network service properly, no matter where it is stored (bsc#1162853).*

Reply:
Much better, thanks